packages/accounts/scripts/plugingen.ts

```diff
@@ -190,10 +200,22 @@ export function plugingen({
       }
 
       const imports = dedent`
-        import { type Address, type GetFunctionArgs, encodeFunctionData } from "viem";
-        import type { Plugin } from "../types";
+        import type {
+          ISmartAccountProvider,
+          SmartAccountProvider,
+          SupportedTransports,
+          UserOperationOverrides,
+        } from "@alchemy/aa-core";
+        import {
+          encodeFunctionData,
+          getContract,
+          type Address,
+          type Chain,
+          type GetFunctionArgs,
+        } from "viem";
+        import { ${contracts[0].name}Abi } from "../../../../plugindefs/index.js";;
```

Comment:
does this work after building? @avasisht23 and I ran into issues importing this Abi like this. We'll probably need to output the Abi by writing it to this file potentially

Reply:
it works. I added the abi exports on index.ts of plugindefs folder

Reply:
does it work even in imported dependencies? there's no definition in the `exports` of the package.json for this project so it won't resolve I'm pretty sure.

we should use `JSON.stringify(contract.abi)` in the plugingen script instead and add the abi to the bottom of plugin.ts file